src/app/public/modules/lookup/lookup.component.scss

```diff
@@ -65,6 +65,12 @@
   }
 }
 
+.sky-lookup-single {
+  sky-tokens.sky-lookup-tokens sky-token {
+    display: none;
```

Comment:
It looks like we're just hiding the tokens, but they're still in the DOM? Can we prevent them from being there in the first place when in single mode?

Reply:
Looking into this one. There are a lot of styling consequences here.... so I'm not sure if we can change it or not.

Reply:
I've done this one. However, I don't love that it requires repeat code in the template.....

The autocomplete has to have something with a directive on it. I would have loved to have just used an `ng-template` but because the main portions of lookup are in an `ng-template` this didn't work as you can't have a template that then pulls in another template (Angular just breaks). So I had to have a little bit of repeat template code to make this work. I also had to add some css to ensure that everything looks consistent when the modes change.